models/api.models.js

```diff
@@ -49,4 +50,28 @@ exports.selectApiArticleComments=(article_id)=>{
     .then(({rows})=>{
       return rows
     })
-}
\ No newline at end of file
+}
+
+
+exports.postArticleComment = (commentToPost, article_id) => {
+    const { username, body } = commentToPost;
+
+    return db
+      .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+      .then(({ rows }) => {
+        if (rows.length === 0) {
+
+          return Promise.reject({ status: 404, msg: "Article id not found" });
+        }
+  
+        return db.query(
+          `INSERT INTO comments (body, votes, author, article_id, created_at)
+           VALUES ($1, 0, $2, $3, NOW())
+           RETURNING *`,
+          [body, username, article_id]
+        );
```

Comment:
Nice use of the sequential logic 🥇  other methods are available as Stephen discussed in the lecture yesterday, like running them simultaneously